client/packages/host/src/components/AppDrawer/SyncNavLink.tsx

```diff
@@ -18,8 +18,17 @@ export const SyncNavLink = () => {
     POLLING_INTERVAL_IN_MILLISECONDS
   );
 
+  // the Badge does not show if the content is 0
+  // somehow though the numberOfRecordsInPushQueue can be '0' which does show
+  const badgeContent = (
+    !numberOfRecordsInPushQueue
+      ? 0
+      : // parse, after casting to string to satisfy ts
+        Number.parseFloat(`${numberOfRecordsInPushQueue}`)
+  ) as React.ReactNode;
+
   const badgeProps = {
-    badgeContent: numberOfRecordsInPushQueue as React.ReactNode,
+    badgeContent,
     max: 99,
```

Comment:
Just out of interest -- what does it show when there is more than 99?

Reply:
just `99+` 😄